### ISSUE TITLE
Dynamic Labels Do Not Default to Name When Field Is Empty

### ISSUE THREAD

**Issue by sqwk** [OFFLINE-GmbH/oc-boxes-support#133]
The following code does not default to `Title & Text `when the `title` field is empty. (like described on https://docs.boxes.offline.ch/concepts/box-configs.html)

```
handle: title-text
name: Title & Text
labelFrom: title
form:
    fields:
        title:
            label: Title
            type: text
```

Boxes v3.6.14 on October v3.6.24.

**Comment by patrick-durrer** [comment]
I was able to reproduce the bug and we will be releasing an update that resolves the issue. 

I will respond to the issue as soon as the releaser is available.

**Comment by tobias-kuendig** [comment]
Thank you for this report @sqwk. The issue is fixed in v3.6.15.
